pkg/bundle/read.go

```diff
@@ -43,14 +44,20 @@ func Open(ctx context.Context, name, baseDir, file string, opts *Options) (*Bund
 	)
 
 	if file == "" {
-		file = filepath.Join(baseDir, "fleet.yaml")
-		if f, err := os.Open(file); os.IsNotExist(err) {
-			in = bytes.NewBufferString("{}")
+		if openedFile, err := os.Open(bundleyaml.GetFleetYamlPath(baseDir, false)); os.IsNotExist(err) {
```

Comment:
Nit: For readbility might be good to wrap in a function and cal it in a loop, like
```
for _, v := range []string{"fleet.yaml". "fleet.yml" {
  in, err := getReader(v)
  if err != nil {
   blabla
  }
  break
}
```

Reply:
Good point. Will fix

Reply:
Alright, I won't use a loop because I want to keep the file close defer statement there, but I have a new refactoring idea I'll push in a moment. Will resolve this once pushed